### ISSUE TITLE
Wrong filename and class name when creating an email class with ace.

### ISSUE THREAD

**Issue by gribbl** [adonisjs/application#34]
### Package version

9.2.1

### Describe the bug

When using the `node ace make:mail verify_email` command, the file generated is as follows:

```bash
app
└── mails
    ├── ...
    └── verify_e_notification.ts
```
And the class name :
```ts
import { BaseMail } from '@adonisjs/mail'

export default class VerifyENotification extends BaseMail {
  // ...
}
```
The mailName generator deletes `mail` and `mailer`. Is this intentional?

**Comment by Julien-R44** [comment]
I believe adding a `removeSuffix('email')` here should be enough : 

https://github.com/adonisjs/application/blob/12258d0ff44e7ca14852612c7294b02731f4bb7e/src/generators.ts#L415

Can you create a PR and adding a test for that please?

**Comment by JeanCastellotti** [comment]
So if I get it right. If we want a verify_email_notification.ts file we would end up with a verify_notification.ts file instead?

**Comment by Julien-R44** [comment]
Yup
You can use the `--intent` parameter to set another suffix

**Comment by thetutlage** [comment]
I think the concern is that our normalizer is from `mail` from the word `email` and hence the file is left with `e` only. Is it correct @JeanCastellotti ?

**Comment by JeanCastellotti** [comment]
Yeah. If someone wants to name his file `verify_email_intent.ts` (like this example in the [documentation](https://docs.adonisjs.com/guides/digging-deeper/mail#class-based-emails)) we end up with `verify_e_intent.ts` or with `verify_intent.ts` with the solution suggested above. But what if the user wants to keep the word `email` in the filename with the default intent? But, this isn't a huge issue, just rename it. :grin: I just wanted to point it out as I have used the ace command recently to generate mail classes.
